napari/_qt/menus/file_menu.py

```diff
@@ -52,13 +53,15 @@ def __init__(self, window: 'Window'):
                     selected=True
                 ),
                 'shortcut': 'Ctrl+S',
+                'enabled': self._layer_count,
```

Comment:
I added a `enabled` key because the `when` is being used to either add or remove an action in a menu, maybe we could change the meaning.

I think when acts more like a enabled disabled not a global on off

Thoughts @tlambert03 ?